packages/dashboard/src/hooks/useWidgetDataTypeSet.ts

```diff
@@ -0,0 +1,21 @@
+import type { SiteWiseAssetQuery } from '@iot-app-kit/source-iotsitewise';
```

Comment:
just calling out the files that matter!
hook to get all the data types of the props in a widget